doc/install.rst

```diff
@@ -221,12 +221,11 @@ command::
     $ sudo port install py36-scikit-learn
 
 
-Canopy and Anaconda for all supported platforms
------------------------------------------------
+Enthought Deployment Manager and Anaconda for all supported platforms
+---------------------------------------------------------------------
 
-`Canopy
-<https://www.enthought.com/products/canopy>`_ and `Anaconda
-<https://www.anaconda.com/download>`_ both ship a recent
+`Enthought Deployment Manager <https://assets.enthought.com/downloads/>`_
+and `Anaconda <https://www.anaconda.com/download>`_ both ship a recent
```

Comment:
The bias continues. Removing the word "recent" more or less deliveries the same message.

```suggestion
`Anaconda <https://www.anaconda.com/download>`_ and
`Enthought Deployment Manager <https://assets.enthought.com/downloads/>`_ both ship a
```